packages/tofu/src/inspectSource.js

```diff
@@ -133,44 +133,38 @@ function inspectGlobals(
   }
 }
 
-function inspectEsmImports(ast) {
-  const esmImports = []
+/**
+ * This finds all modules `import`ed into the AST, as well as any re-exported modules.
+ *
+ * @param {import('@babel/types').Node} ast
+ * @param {string[]} [packagesToInspect]
+ * @returns {{ esmImports: string[] }}
+ */
+function inspectEsmImports(ast, packagesToInspect) {
```

Comment:
The previous implementation of this function returned "nested" imports where appropriate, e.g., `import {foo} from 'bar'` results in `bar.foo`.  This does not happen in CJS packages, so I figured it best to maintain parity. 

 I'm unsure of any drawback to retaining the old behavior except that it will likely explode the size of a generated policy file.

Reply:
Also, we don't actually need the `switch` statement nor the extra conditionals--these look to have been purely defensive checks--based on the documented type of the `NodePath` as provided to the callbacks to `traverse()`